frsca.cue

```diff
@@ -98,3 +120,59 @@ frsca: clusterPolicy?: [_]: {
         }]
     }]
 }
+
+// watch is used to add a tekton trigger to run a pipeline
+frsca: trigger?: [Name=_]: {
+	pipelineRun: pipelineV1Beta1.#PipelineRun & {
+		apiVersion: "tekton.dev/v1beta1"
+		kind:       "PipelineRun"
+		metadata: {
+			generateName: *"\(Name)-" | string
+			labels: "app.kubernetes.io/description": "PipelineRun"
```

Comment:
can include `namespace: $(tt.params.namespace)` based on the comment below regarding adding a default namespace parameter to the `triggerTemplate`.